.github/workflows/sam-smoke-tests.yaml

```
@@ -1,94 +1,27 @@
 name: sam-smoke-tests
 
 on:
-  pull_request:
-    paths-ignore: ['**.md']
-  push:
-    branches:
-      - develop
-    paths-ignore: ['**.md']
+  workflow_call:
+    inputs:
+      sam-hostname:
+        description: 'a generic hostname for Sam'
+        required: true
+        type: string
+      ENV:
+        description: 'test environment (qa, alpha)'
```

```suggestion
        description: 'test environment (dev, staging, qa, alpha)'
```
I think we should either list all possible environments here, or none.

---

@jyang-broad suggested dropping this file since we already have `sam-smoke-tests.yaml` [here](https://github.com/broadinstitute/terra-github-workflows/blob/main/.github/workflows/sam-smoke-tests.yaml) so will change the description there instead.